op-scim-bridge/templates/deployment.yaml

```diff
@@ -63,6 +63,10 @@ spec:
           volumeMounts:
             - mountPath: /home/opuser/.op
               name: {{ tpl .Values.scim.name . }}-credentials
+          {{ with .Values.scim.init.resources }}
```

Comment:
```suggestion
          {{ with .Values.scim.initResources }}
```

Reply:
I think we unfortunately unintentionally removed the config from the `deployment.yaml`, would you mind adding it back?

I believe it should be:
```yaml
          {{ with .Values.scim.initContainers.resources }}
          resources:
            {{- toYaml . | nindent 12 }}
          {{ end }}
```

Reply:
I am sorry, it was my fault. I'll try to get it back, I am a little confused by the flow of commits to commits.

Reply:
No problem at all. Thanks for fixing it.